lang/en/theme_boost_union.php

```diff
@@ -100,7 +100,8 @@
 $string['loginbackgroundimagesheading'] = 'Login page background images';
 // ... ... Setting: Login page background image.
 $string['loginbackgroundimage'] = 'Login page background images';
-$string['loginbackgroundimage_desc'] = 'The images to display as a background of the login page. One of these images will be picked randomly and shown when the user visits the login page.';
+$string['loginbackgroundimage_desc'] = 'The images to display as a background of the login page. One of these images will be picked randomly and shown when the user visits the login page.
+Please make sure not to use non-ASCII-characters in the filename if you want to display text for login background images.';
```

Comment:
You have added a line break within a string. This does not directly result in a line break on the settings page. You have to include a `<br>` tag into the string to trigger a line break. 
However, I would propose to remove the line break entirely and just append to new string to the existing string.